### ISSUE TITLE
Empty lines in new terminal

### ISSUE THREAD

**Issue by proksi21** [dylanaraps/pywal#306]
Steps to reproduce:

- execute `wal -i "image_name" -g`
- add `(cat ~/.cache/wal/sequences &)` to your .bashrc

What happens:

- Empty lines appears before command prompt (see picture)
![screenshot at 2018-09-27 13-02-44](https://user-images.githubusercontent.com/33513568/46134166-7598ba00-c26c-11e8-9b01-6d3bc675cd26.png)

**Comment by mpolidori** [comment]
I think this is the same as #304.

**Comment by pycebs** [comment]
I agree with @mpolidori 

What terminal are you using?

**Comment by proksi21** [comment]
@pycebs I'm using mate-terminal.
If I comment `cat ~/.cache/wal/sequences` - everything is ok (but without colors, obviously)
Here's my sequences file
[sequences.zip](https://github.com/dylanaraps/pywal/files/2426760/sequences.zip)

**Comment by mpolidori** [comment]
@proksi21 In `sequences.py`, try commenting out:
 ```
sequences.extend([
     "\033[s\033[1000H\033[8m%s\033[u" %
     set_special(708, colors["special"]["background"], "h", alpha),
     set_special(13, colors["special"]["cursor"], "l")
])
```

**Comment by dessalines** [comment]
That seemed to work for me.

**Comment by proksi21** [comment]
@mpolidori that helped, thanks.
Alternatively, one can add `clear` at the end of .bashrc, that also helps.

**Comment by dylanaraps** [comment]
Will fix this.

**Comment by deviantfero** [comment]
@dylanaraps shouldn't this be fixed with `3.2.0` and the VTE sequence removal?

**Comment by dylanaraps** [comment]
yes
